NineChronicles.Headless/GraphTypes/ActionQuery.cs

```diff
@@ -347,6 +349,56 @@ public ActionQuery(StandaloneContext standaloneContext)
                     return Encode(context, action);
                 }
             );
+
+            Field<NonNullGraphType<ByteStringType>>(
+                "runeEnhancement",
+                arguments: new QueryArguments(
+                    new QueryArgument<NonNullGraphType<AddressType>>
+                    {
+                        Name = "avatarAddress",
+                        Description = "The avatar address to receive staking rewards."
+                    },
+                    new QueryArgument<NonNullGraphType<IntGraphType>>
+                    {
+                        Name = "runeId",
+                        Description = "The ID of rune to make or enhance"
+                    },
+                    new QueryArgument<IntGraphType>
+                    {
+                        Name = "tryCount",
+                        Description = "The try count to create or enhance this rune."
+                    }
+                ),
+                resolve: context =>
+                {
+                    var service = standaloneContext.NineChroniclesNodeService;
+                    if (service is null)
+                    {
+                        Log.Error($"{nameof(NineChroniclesNodeService)} is null.");
+                        return null;
+                    }
```

Comment:
Because the type of `runeEnhancement` is a *non-null* type, it will cause errors. How about throwing errors instead?